test/setup/gaia-setup.sh

```diff
@@ -15,12 +17,23 @@ if [ -z "$2" ]; then
   exit 1
 fi
 
+if [ -z "$3" ]; then
+  echo "Need to input seeds of validator key..."
+  exit 1
+fi
+
```

Comment:
the docker image doesn't need the seed. The seed will be referenced in the go test. The docker image only needs the correct priv key